### DISCUSSION TITLE
Anzeige der 3 Phasen von Hoymiles möglich

### DISCUSSION THREAD

**Discussion by Zurrmaxe** [hoylabs/OpenDTU-OnBattery#940] [Q&A]
Hallo zusammen
Ich habe einen Hoymiles 2260 mit 3 Phasen. 
Kann man sich die 3 Phasen einzeln anzeigen lassen ?
Konnte diesbezüglich im Netz noch nichts finden.

**Comment by spcqike** [comment]
https://github.com/tbnobody/OpenDTU/discussions/1303

Bisher scheinbar nicht.

**Comment by Zurrmaxe** [comment]
Die nächste Frage wäre, ob der Hoymiles die 3 Phasen überhaupt getrennt zur Verfügung stellt und dann von der DTU ausgelesen könnte.
